components/ai_chat/core/browser/ai_chat_metrics_unittest.cc

```diff
@@ -204,4 +240,11 @@ TEST_F(AIChatMetricsUnitTest, OmniboxWeekCompare) {
   histogram_tester_.ExpectTotalCount(kOmniboxWeekCompareHistogramName, 14);
 }
 
+TEST_F(AIChatMetricsUnitTest, Reset) {
+  ai_chat_metrics_->RecordReset();
+  histogram_tester_.ExpectUniqueSample(kAcquisitionSourceHistogramName,
+                                       INT_MAX - 1, 1);
+  histogram_tester_.ExpectUniqueSample(kEnabledHistogramName, INT_MAX - 1, 1);
+}
```

Comment:
Can we use `std::numeric_limits<int>::max())` instead of INT_MAX, so that we can prefer stl to macro sytle?